packages/oas/src/lib/openapi-to-json-schema.ts

```diff
@@ -322,19 +322,11 @@ export function toJSONSchema(data: RMOAS.SchemaObject | boolean, opts: toJSONSch
               return arr;
             },
 
-            // JSON Schema ony supports examples with the `examples` property, since we're
-            // ingesting OpenAPI definitions we need to add a custom resolver for its `example`
-            // property.
-            example: (obj: unknown[]) => obj[0],
```

Comment:
we no longer need to specify these keywords since they all the did the same thing as our `defaultResolver` 🎊